src/registrar/models/domain_application.py

```diff
@@ -474,6 +490,28 @@ def show_organization_federal(self) -> bool:
         user_choice = self.organization_type
         return user_choice == DomainApplication.OrganizationChoices.FEDERAL
 
+    def show_tribal_government(self) -> bool:
+        """Show this step if the answer to the first question was "tribal"."""
+        user_choice = self.organization_type
+        return user_choice == DomainApplication.OrganizationChoices.TRIBAL
+
+    def show_tribal_explanation(self) -> bool:
+        """Show this step if the tribe is not federally or state recognized."""
+        user_choice = self.organization_type
+        if user_choice == DomainApplication.OrganizationChoices.TRIBAL:
+            # did answer tribal, check the recognition answers
+            if (
+                self.federally_recognized_tribe is not None
+                and self.state_recognized_tribe is not None
+            ):
+                # have answered these questions
+                if (
+                    not self.federally_recognized_tribe
+                    and not self.state_recognized_tribe
+                ):
+                    return True
+        return False
```

Comment:
If I properly understand the logic here, it could be reduced to:

```suggestion
            # did answer tribal, check the recognition answers
            return (
                self.federally_recognized_tribe == False
                and self.state_recognized_tribe == False
            )
```

(but I have not tested this)